migration/1701689359018-add_fields_to_qf_round_history.ts

```diff
@@ -0,0 +1,69 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class addFieldsToQfRoundHistory1701689359018
+  implements MigrationInterface
+{
+  async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+            ALTER TABLE "qf_round_history"
+            ADD COLUMN "matchingFundAmount" real ,
+            ADD COLUMN "matchingFundPriceUsd" real ,
+            ADD COLUMN "matchingFundCurrency" text ;
+        `);
+
+    await queryRunner.query(`
+            ALTER TABLE "donation"
+            ADD COLUMN "distributedFundQfRoundId" integer;
+
+            -- If you have a foreign key constraint to enforce the relationship
+            ALTER TABLE "donation"
+            ADD CONSTRAINT "FK_donation_qfRound"
+            FOREIGN KEY ("distributedFundQfRoundId") REFERENCES "qf_round"("id");
+        `);
+
+    // These rounds are in Production but I didnt set any condition for that
+    // because I want this part of code be executed in staging ENV
+
+    // Alpha round in production
+    await queryRunner.query(`
+        UPDATE qf_round_history 
+        SET 
+            "matchingFundAmount" = "matchingFund", 
+            "matchingFundPriceUsd" = 1, 
+            "matchingFundCurrency" = 'WXDAI'
+        WHERE 
+            id = 2 AND "matchingFund" IS NOT NULL;
```

Comment:
why are we hard coding the id to 2?

Reply:
We have some qf rounds already in our DB and they have qfRoundHistory items in DB, they dont have new fields, so I had to put their ids hard coded here